src/PayPal/Api/Plan.php

```diff
@@ -0,0 +1,128 @@
+<?php
+declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Swag\PayPal\PayPal\Api;
+
+use Swag\PayPal\PayPal\Api\Common\PayPalStruct;
+use Swag\PayPal\PayPal\Api\Plan\BillingCycle;
+use Swag\PayPal\PayPal\Api\Plan\PaymentPreferences;
+use Swag\PayPal\PayPal\Api\Plan\Taxes;
+
+class Plan extends PayPalStruct
+{
+    /** @var string */
```

Comment:
please use three lines for the annotation to be consistent with the other structs